src/main/java/life/qbic/data/processing/registration/ProcessRegistrationRequest.java

```diff
@@ -40,53 +52,134 @@ public class ProcessRegistrationRequest extends Thread {
   private final ConcurrentRegistrationQueue registrationQueue;
   private final Path workingDirectory;
   private final Path targetDirectory;
+  private final String metadataFileName;
+  private final Path usersErrorDirectory;
```

Comment:
Major NIT, since we only have one user
```suggestion
  private final Path userErrorDirectory;
```